README.md

```diff
@@ -128,3 +118,5 @@ This project is [MIT](./LICENSE) licensed.
 _NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
Great job @vvoo21 🤝

- [ ] [Optional] Kindly remove this part from your license section. It is not necessary when the license has already been created. This will make your readme file more professional.
![readme](https://user-images.githubusercontent.com/63917725/232260157-6774da8e-40bf-4eb8-8639-508f26ae0cf2.PNG)